go.mod

```diff
@@ -10,3 +10,5 @@ require (
 	golang.org/x/net v0.0.0-20191007182048-72f939374954
 	google.golang.org/grpc v1.24.0
 )
+
+replace github.com/fiorix/go-diameter/v4 => ./
```

Comment:
without this, all internal imports are actually using the code from the original library (i.e. github.com/fiorix/go-diameter/v4). it's funny we survived so long without this.